client/src/pages/Chart.tsx

```diff
@@ -0,0 +1,251 @@
+        const uniqueData: GrossMevDataResponse[] = [];
+        for (let i = 0; i < sortedData.length; i++) {
+          if (i === 0 || sortedData[i].time !== sortedData[i - 1].time) {
+            uniqueData.push(sortedData[i]);
+          }
+        }
+
+        // Keep only the last 60 data points
+        setGrossMevData(uniqueData.slice(-60));
+      } catch (error) {
+        console.error('Error fetching messages from PubNub:', error);
+      }
+    })();
+
+    // Cleanup on unmount
+    return () => {
+      pubnub.removeListener(listener);
+      pubnub.unsubscribe({
+        channels: [PUBNUB_CHANNELS.GROSS_MEV],
+      });
+    };
+  }, []);
+
+  // Transform sample data into chart format
+  const chartData = {
+    labels: grossMevData.map(item => {
+      const date = new Date(item.time * 1000);
+      return `${date.getHours().toString().padStart(2, '0')}:${date.getMinutes().toString().padStart(2, '0')}`;
+    }),
+    datasets: [
+      {
+        label: 'Total MEV Profit (USD)',
+        data: grossMevData.map(item => item.total),
+        borderColor: 'rgb(75, 192, 192)',
+        backgroundColor: 'rgba(75, 192, 192, 0.2)',
+        tension: 0.1,
+        fill: true,
+      },
+      {
+        label: 'Normal MEV Profit (USD)',
+        data: grossMevData.map(item => item.normal),
+        borderColor: 'rgb(54, 162, 235)',
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        tension: 0.1,
+        fill: true,
+      },
+      {
+        label: 'Timeboost MEV Profit (USD)',
+        data: grossMevData.map(item => item.timeboost),
+        borderColor: 'rgb(255, 99, 132)',
+        backgroundColor: 'rgba(255, 99, 132, 0.2)',
+        tension: 0.1,
+        fill: true,
+      },
+    ],
+  };
+
+  const options = {
+    responsive: true,
+    maintainAspectRatio: false,
+    plugins: {
+      legend: {
+        position: 'top' as const,
+      },
+      title: {
+        display: true,
+        text: 'MEV Profit Over Time',
+      },
+      tooltip: {
+        enabled: true,
+        mode: 'index' as const,
+        intersect: false,
+        callbacks: {
+          title: (context: any) => {
+            const index = context[0]?.dataIndex;
+            if (index === undefined) return '';
+            const item = grossMevData[index];
+            if (!item) return '';
+            const date = new Date(item.time * 1000);
+            return `${date.toLocaleDateString()} ${date.getHours().toString().padStart(2, '0')}:${date.getMinutes().toString().padStart(2, '0')}`;
+          },
+          label: (context: any) => {
+            const label = context.dataset.label || '';
+            const value = context.parsed.y;
+            return `${label}: $${value.toLocaleString('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 })}`;
+          },
+        },
+      },
+    },
+    scales: {
+      y: {
+        beginAtZero: true,
+      },
+    },
+  };
+
+  return (
+    <div className="chart-container">
```

Comment:
let's not use inline styles. move all css related stuff to the associated .css file